### PR TITLE
Add pyscreeze

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

File: recipes/pyscreeze/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pillow
```
Comment (synapticarbors) [inline recipes/pyscreeze/meta.yaml]: it doesn't look like `pillow` is actually required to build the package.

File: recipes/pyscreeze/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+package:
+  name: pyscreeze
+  version: {{ version }}
+
+source:
+  fn: PyScreeze-{{ version }}.tar.gz
+  md5: 5ce611432f38db75b341eec54428530f
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+
+build:
```
Comment (synapticarbors) [inline recipes/pyscreeze/meta.yaml]: Consider making `noarch: python`?

File: recipes/pyscreeze/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
+{% set version = "0.1.13" %}
+{% set name = "PyScreeze" %}
+
+package:
+  name: pyscreeze
+  version: {{ version }}
+
+source:
+  fn: PyScreeze-{{ version }}.tar.gz
+  md5: 5ce611432f38db75b341eec54428530f
```
Comment (synapticarbors) [inline recipes/pyscreeze/meta.yaml]: `sha256: 4d752af522fcae606e56b77091fc27376fc0a3711a7ee8855b1f24f084b12783`

File: recipes/pyscreeze/meta.yaml (hunk 1 above)
Comment (jschueller) [reply recipes/pyscreeze/meta.yaml]: ok, done

**Pull request merged**